### DISCUSSION TITLE
v19.09.4-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#87] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/qnblackcat/uYouPlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.09.4`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.09.4-3.0.1:** (2024-03-07)

- Updated Youtube from `19.09.3` to `19.09.4`
- Updated YTLite from `3.0` to `3.0.1`
    - Fixed random crashes when auto-skip shorts option enabled
  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v16.46.5-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*
- Hiding Shorts creates an empty space instead #55

<hr /><em>This discussion was created from the release <a href='https://github.com/Balackburn/YTLitePlus/releases/tag/v19.09.4-3.0.1'>v19.09.4-3.0.1 - YTLitePlus</a>.</em>

**Comment by shahin365** [comment]
can you please upload it somewhere else? this 'build it yourself' is too technical. I'm not tech-savvy. Thanks.

**Reply by Balackburn** [reply]
"Too technical" their is 3 steps really easy anybody can do it

**Reply by egrandblvd** [reply]
help!

**Reply by egrandblvd** [reply]
Thanks...For the archive!!! :) 👍

**Reply by abheizthe** [reply]
thanks, succesfully build it!!

**Reply by abheizthe** [reply]
sorry but, it's possible to delete some files (tweaks) that i didn't used it from forked repo? and how to do it?

i'm always deleting some tweaks from the app bundle after the app was installed
thank you

**Reply by abheizthe** [reply]
i can't built it, please take a look. thank you

![yt](https://github.com/YTLitePlus/YTLitePlus/assets/54247318/175d3b7d-dd6f-418c-9989-1d0ccd46a33e)

**Reply by Balackburn** [reply]
Fixed

**Comment by Balackburn** [comment]
It’s archived here on archive.org if you want :
 
https://archive.org/details/ytlite-plus-19.05.3-2.4.1

(All the versions are archived here)

**Reply by shahin365** [reply]
Awesome! Thank you 👍

**Reply by shahin365** [reply]
Archive.org link doesn't work anymore!

**Reply by Balackburn** [reply]
It got DCMAed too

**Comment by egrandblvd** [comment]
Thanks 🙏!Sent from my iPhoneOn Mar 11, 2024, at 8:21 AM, abheizthe ***@***.***> wrote:﻿
thanks, succesfully build it!!
